.github/workflows/smoke_test.yml

```diff
@@ -1,6 +1,8 @@
 name: Smoke Test
 
 on:
+  schedule:
+    - cron: '*/5 * * * *'
```

Comment:
is every 5 minutes overkill?

Reply:
Hmm, maybe I should remove this all-together for time being.